src/commands/git_secret_hide.sh

```diff
@@ -160,11 +160,17 @@ function hide {
   
       # encrypt file only if required
       if [[ "$fsdb_file_hash" != "$file_hash" ]]; then
+
+        set +e   # disable 'set -e' so we can capture exit_code
```

Comment:
We disable 'set -e' here and re-enable it below.

Reply:
Yes, looks fine!